src/sentry/search/eap/spans/aggregates.py

```diff
@@ -487,9 +487,149 @@ def resolve_bounded_sample(args: ResolvedArguments) -> tuple[AttributeKey, Trace
             AttributeArgumentDefinition(
                 attribute_types={
                     "string",
+                    "number",
+                    "integer",
                 },
                 default_arg="log.body",
             )
         ],
     ),
+    "sum": AggregateDefinition(
+        internal_function=Function.FUNCTION_SUM,
+        default_search_type="duration",
+        arguments=[
+            AttributeArgumentDefinition(
+                attribute_types={
+                    "duration",
+                    "number",
+                    "integer",
+                    *constants.SIZE_TYPE,
+                    *constants.DURATION_TYPE,
+                },
+            )
+        ],
+    ),
+    "avg": AggregateDefinition(
+        internal_function=Function.FUNCTION_AVG,
+        default_search_type="duration",
```

Comment:
These maybe should be `number` since we're not defaulting to `span.duration` and most attributes will be numbers I suspect?